lib/ASRouterTargeting.jsm

```diff
@@ -308,6 +308,9 @@ const TargetingGetters = {
 
     return false;
   },
+  get hasAccessedFxAPanel() {
+    return Services.prefs.getBoolPref("identity.fxaccounts.toolbar.accessed", true);
```

Comment:
What's your rationale behind returning `true` by default here? To prevent false positives? I think that makes sense for the for this case but it might not be for other usages